### PR TITLE
Kubernetes web app template page

### PR DESCRIPTION
**Pull request by gthuang**
Adding landing page for new k8s deployment+service template.

### CODE REVIEW

**Comment by gthuang** [comment]
@cnunciato how do we change the right-side section that says "This template is also available for:"? It currently lists the kubernetes cluster templates, but it doesn't make sense to do that for this page.

File: themes/default/content/templates/kubernetes/webapp/index.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+When the deployment completes, Pulumi exports the following [stack output](/docs/intro/concepts/stack#outputs) values:
+
+deploymentName
+: The name of your new Kubernetes Deployment.
+
+serviceName
+: The name of your new Kubernetes Service.
+
+Output values like these are useful in many ways, most commonly as inputs for other stacks or related cloud resources. The computed `serviceName`, for example, can be used from the command line to open the newly deployed web application in your favorite web browser:
```
Comment (scottslowe) [inline themes/default/content/templates/kubernetes/webapp/index.md:55]: Unfortunately, you can't use the computed `serviceName` output in this way. Although it is exposed as a Service, it's not consumable using that URL.

File: themes/default/content/templates/kubernetes/webapp/index.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+Projects created with the Kubernetes Application template expose the following [configuration](/docs/intro/concepts/config) settings:
+
+namespace
+: The name of the namespace to be created in your existing cluster. Defaults to `default`.
+
+replicas
+: The number of replicated Pods to be created in your new Deployment. Defaults to `1`.
+
+webserverconfig
```
Comment (scottslowe) [inline themes/default/content/templates/kubernetes/webapp/index.md:71]: This isn't exposed as a configuration setting.

File: themes/default/content/templates/kubernetes/webapp/index.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+: The ConfigMap that stores your application's configuration. By default, it is configured for Nginx.
+
+webserverdeployment
```
Comment (scottslowe) [inline themes/default/content/templates/kubernetes/webapp/index.md:74]: Same here---this isn't exposed as a configuration setting.

**Review by scottslowe — CHANGES_REQUESTED** [review]
Looks great overall, just a few minor changes needed. Thanks for putting this together!

**Comment by scottslowe** [comment]
@gthuang Regarding the use of `serviceName`: we _could_ put in the text that users could port-forward to that Service for testing/verification. I can provide more details if you all think that might be helpful.

File: themes/default/content/templates/kubernetes/webapp/index.md [hunk 4]
```diff
@@ -52,28 +52,18 @@ deploymentName
...
-
-```bash
-$ open $(pulumi stack output serviceName)
-```
+Output values like these are useful in many ways, most commonly as inputs for other stacks or related cloud resources.
 
 ## Customizing the project
 
-Projects created with the Kubernetes Application template expose the following [configuration](/docs/intro/concepts/config) settings:
+You can change the behavior of the Deployment by passing values to it and an example is already included in the template. Projects created with the Kubernetes Application template expose the following [configuration](/docs/intro/concepts/config) settings:
```
Comment (scottslowe) [inline themes/default/content/templates/kubernetes/webapp/index.md:59]: I don't think I agree with this addition as it isn't really accurate.

**Review by scottslowe — CHANGES_REQUESTED** [review]
One minor nit regarding an addition, but LGTM otherwise

**Review by scottslowe — APPROVED** [review]

**Review by cnunciato — APPROVED** [review]

**Pull request merged**